deploy/platform-cr.yaml

```diff
@@ -8,3 +8,4 @@ spec:
   build:
     camelVersion: "2.23.1"
     baseImage: "fabric8/s2i-java:3.0-java8"
+    localRepository: "/tmp/artifacts/m2"
```

Comment:
Better leaving it blank here and providing a default when initializing the platform. There are platforms that will not be created by `kamel install` and don't contain this option. Same is for the `baseImage` and also `camelVersion`, do they have a sensible default?

When initializing the platform, we may check for empty values and pick them from this resource file. Wdyt?

Reply:
I do agree, what about opening a new issue for the defaults ?

Reply:
Ok, it seems common to other props